nestkernel/conn_builder.h

```diff
@@ -105,7 +109,19 @@ class ConnBuilder
 
   bool all_parameters_scalar_() const;
 
-  bool change_connected_synaptic_elements( index, index, const int, int );
+/**
+ * Updates the number of connected synaptic elements in the
+ * target and the source.
+ * Returns 0 if the target is either on another
+ * MPI machine or another thread. Returns 1 otherwise.
+ *
+ * @param snode_id id of the source
+ * @param tnode_id id of the target
+ * @param tid thread id
+ * @param update amount of connected synaptic elements to update
+ * @return
+ */
+  bool change_connected_synaptic_elements( index snode_id, index tnode_id, const int thread tid, int update );
```

Comment:
This should fix the build problems:

```suggestion
  bool change_connected_synaptic_elements( index snode_id, index tnode_id, const int tid, int update );
```